### PR TITLE
Auto inject CRI-O version

### PR DESCRIPTION
**Pull request by saschagrunert**
We now use a new tool to automatically discover either:

- The current tag if it is checked out.
- The next development version (`-dev`) for master (next minor) or
  release (next patch) branches.

In case the script reports "unknown", then it is now still possible to
inject the VERSION via `make VERSION=my-new-version`.

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--additional-devices**="": Devices to add to the containers  (default: [])
 
-**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is 'crio-default-' followed by the version string of CRI-O. (default: "crio-default-1.17.0-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod's metadata annotation.
+**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is 'crio-default-' followed by the version string of CRI-O. (default: "crio-default-1.18.0-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod's metadata annotation.
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: IDK how, but it would sure be nice to automatically generate the version here in these files.

Comment (saschagrunert) [reply docs/crio.8.md]: I would like to get rid of this AppArmor version from my personal preference, it's too error prone. I'll follow up in a dedicated PR.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by saschagrunert** [comment]
Rebased on top of the latest master to fix the CI issues

**Comment by TomSweeneyRedHat** [comment]
Happy green tests on this one too, needs an /lgtm

**Comment by rhatdan** [comment]
@giuseppe I though e2e_crun tests were not blocking

**Comment by cgwalters** [comment]
I'm pretty sure it was this PR that broke OpenShift 4 builds:
https://bugzilla.redhat.com/show_bug.cgi?id=1825623

**Pull request merged**